vim.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:
+Frame        Function    Args
+00180238000  0018005D19E (00180223639, 00180223C39, 001802342F0, 000FFFFB720)
```

Comment:
Needs license header 😸

Reply:
Actually you could add `.exe.stackdump` to our `.gitignore` at the root.

Reply:
oh no! didn't mean to add this :)